buckwheat/tests/test_tokenizer.py

```diff
@@ -5,95 +5,149 @@
+                          "/* Calculating area of Rectangle.\nCreate"
+                          " function clacArea.\nArea calculations"
+                          " = height * width.\n */"]],
                         ["Python", "test.py",
                          {"tree": 6, "sitter": 6, "dir": 4, "get": 3, "path": 3, "parser": 2,
                           "str": 2, "bin": 2, "loc": 2, "languag": 1, "abspath": 1, "dirnam": 1,
-                          "file": 1, "join": 1}],
+                          "file": 1, "join": 1},
+                         ["# No parsers need to be declared here"]],
                         ["Java", "test.java",
                          {"stats": 7, "sub": 7, "dir": 4, "writer": 3, "file": 3, "dirs": 3,
                           "now": 3, "statist": 3, "all": 3, "collect": 2, "write": 2, "list": 2,
                           "featur": 2, "total": 2, "holder": 2, "and": 1, "print": 1, "string": 1,
                           "except": 1, "ioexcept": 1, "length": 1, "directori": 1,
                           "isdirectori": 1, "system": 1, "out": 1, "println": 1, "get": 1,
                           "name": 1, "collector": 1, "from": 1, "project": 1, "path": 1,
-                          "topath": 1, "size": 1, "min": 1, "files": 1, "test": 1, "totest": 1}],
+                          "topath": 1, "size": 1, "min": 1, "files": 1, "test": 1, "totest": 1},
+                         ["/* Function that collects and writes statistics.subDirs\n"
+                          "    Uses StatisticsHolder class for statistics information.\n    */",
+                          "// total number of subDirs"]],
                         ["Go", "test.go",
                          {"buff": 4, "file": 4, "fmt": 3, "fprintln": 3, "err": 3, "out": 2,
                           "string": 2, "name": 2, "languag": 2, "version": 1, "commit": 1,
                           "print": 1, "break": 1, "down": 1, "buffer": 1, "data": 1, "read": 1,
-                          "limit": 1}],
+                          "limit": 1},
+                         ["// Declaration of variables", "/* Breaking Down function\n   iterating over name and language\n*/"]],
                         ["C++", "test.cpp",
                          {"argc": 3, "argv": 3, "mini": 3, "maxi": 3, "std": 2, "atof": 2,
                           "acosd": 2, "main": 1, "zero": 1, "valmax": 1, "exhaust": 1, "test": 1,
-                          "pack": 1, "raw": 1}],
+                          "pack": 1, "raw": 1},
+                         ["// Initialize mini with zero", "/* Initialize maxi with maxval */"]],
                         ["Ruby", "test.rb",
                          {"out": 4, "tmp": 4, "string": 4, "get": 2, "block": 2, "new": 2,
-                          "stdout": 1, "stderr": 1}],
+                          "stdout": 1, "stderr": 1},
+                         ["# Declaration of get_stdout function", "=begin\nDeal with stderr\n=end"]],
```

Comment:
Is it possible to extract comments without `/`, `#`, etc?

Reply:
I haven't find any solution for this task neither in tree-sitter nor in pygments. I think it may be done using postprocessing after extracting comments using `get_comments_from_file` method for different cases differently.